documentation/general/dash-sonic-hld.md

```diff
@@ -265,6 +266,73 @@ It is possible that a given packet can get a hit in route table and/or mapping t
 
   ![dash-outbound-meter](./images/dash-hld-outbound-meter-pipeline.png)
 
+## 2.5 FastPath
+
+This section captures the Sonic-Dash specifics of FastPath use-case. Detailed document on FastPath is captured here (![FastPath](https://github.com/sonic-net/DASH/blob/main/documentation/load-bal-service/load-balancer-v3.md))
+The following are the salient points and requirements:
+- FastPath kicks in when appliance receives an ICMP redirect that matches an existing unified flow
+- ICMP redirects are expected to be received from source and destination MUXes separately
+- Each ICMP redirect shall only update one side of the flow. (Src or Dst depending on the originating MUX)
```

Comment:
"(Src or Dst depending on the originating MUX)"

from the comment for the point above, we might be able to emit this part.

Reply:
done